src/styles/AllocationOverview.scss

```diff
@@ -0,0 +1,20 @@
+.AllocationOverview {
+
+}
+
+@media screen and (min-width: $sm) {
+    .AllocationOverview {
+        .edit-delete {
+            width: 70%;
+        }
+    }
+}
+
+@media screen and (max-width: $sm){
+    .AllocationOverview {
```

Comment:
I don't get why this media query is needed, you don't get the same result moving this to the `.AllocationOverview` at the top of the file? since `@media screen and (min-width: $sm)` is checking for bigger screens whatever you put outside the media query at `. AllocationOverview` should be applied in the cases where `max-width: $sm`. Pls check this and make sure it's not redundant code.

Reply:
Ohhhh ok, I see I was doing it wrong! Going to fix it, ty!